configs/openxr_app.yaml

```diff
@@ -2,5 +2,8 @@ app:
   src_path:
     git_repo : https://gitlab.freedesktop.org/monado/demos/openxr-simple-example
     version  : master
-  bin_subpath: build/openxr-example
+  # bin_subpath: Materials_VR.x86_64
+  bin_subpath: Sponza_VR.x86_64
```

Comment:
Should we stick with openxr simple example since we want to run it out of the box?

Reply:
yes, will update.